backend/framework/bin/scalar_metrics.py

```diff
@@ -41,7 +41,14 @@ class LoadMetrics:
                , ['SKYCONT_ALARM_RANGE', 'SKYCONT_WARN_RANGE', 'B_CONT', 'Z_CONT', 'R_CONT']
                , ['B_PEAKS', 'R_PEAKS', 'SUMCOUNT_WARN_RANGE', 'SUMCOUNT_ALARM_RANGE', 'Z_PEAKS']
                , ['PCHI_RESID', 'PER_RESID', 'SKY_ALARM_RANGE', 'SKY_WARN_RANGE', 'BIN_SZ']
-               , ['FIDSNR_WARN_RANGE', 'FIDSNR_ALARM_RANGE', 'FIDMAG']]
+               , ['FIDSNR_WARN_RANGE', 'FIDSNR_ALARM_RANGE',
+                  'FIDMAG', 'ELG_FITRESULTS',
+                  'LRG_FITRESULTS', 'QSO_FITRESULTS',
+                  'STAR_FITRESULTS', 'ELG_SNR_MAG',
+                  'ELG_FIBERID', 'LRG_SNR_MAG',
+                  'LRG_FIBERID', 'QSO_SNR_MAG',
+                  'QSO_FIBERID', 'STAR_SNR_MAG',
+                  'STAR_FIBERID', 'RA', 'DEC']]
```

Comment:
I'm not sure if the keys from 'ELG_FITRESULTS' are necessary ( If they are, I am fine!).

Reply:
Is used here:

https://github.com/desihub/qlf/pull/87/files#diff-0abaa40084a9cd279504e084b2f80e9bR110